sql/string-functions.md

```diff
@@ -0,0 +1,74 @@
+| [`LEFT()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_left)                           | 返回指定的最左字符                                                                                     |
+| [`RIGHT()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_right)                         | 返回指定的最右字符                                                                                       |
+| [`INSTR()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_instr)                         | 返回子字符串的第一个出现位置                                                                                      |
+| [`LOCATE()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_locate)                       | 返回子字符串的第一个出现位置，与 `INSTR()` 的参数位置相反                                                                                   |
+| [`POSITION()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_position)                   | 与 `LOCATE()` 功能相同                                                                                                                      |
+| [`REPEAT()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_repeat)                       | 返回重复指定次数的字符串                                                                                             |
+| [`CONCAT()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_concat)                       | 返回连接的字符串                                                                                                                |
+| [`CONCAT_WS()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_concat-ws)                 | 返回由分隔符连接的字符串                                                                                                         |
+| [`REVERSE()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_reverse)                     | 返回和字符顺序相反的字符串                                                                                                        |
+| [`SPACE()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_space)                       | 返回指定数目的空格组成的字符串                                                                                          |
+| [`FIELD()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_field)                         | 返回参数在后续参数中出现的第一个位置                                                             |
+| [`ELT()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_elt)                             | 返回指定位置的字符串                                                                                                             |
+| [`EXPORT_SET()`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_export-set)               | 返回一个字符串，其中对于 bits 值中的每个位组，可以得到一个 on 字符串，而对于每个清零比特位，可以得到一个 off 字符串        |
```

Comment:
Return a string such that for every bit (set in the value bits), you get an on string and for every unset bit, you get an off string

here, is "set in the value bits" a postpositive attributive? @queenypingcap